test/test_graph_generators.py

```diff
@@ -1,13 +1,13 @@
 import os
 from rdflib import Graph, URIRef
 
-michel = URIRef("urn:michel")
-tarek = URIRef("urn:tarek")
-bob = URIRef("urn:bob")
-likes = URIRef("urn:likes")
-hates = URIRef("urn:hates")
-pizza = URIRef("urn:pizza")
-cheese = URIRef("urn:cheese")
+michel = URIRef("urn:x-rdflib:michel")
```

Comment:
While `urn:x-rdflib:michel` is significantly better than `urn:michel`, we should rather be using example URN namespace `urn:example:` [[IETF RFC 6963](https://datatracker.ietf.org/doc/html/rfc6963)]  or even the `example:` URI scheme [[IETF RFC 7595](https://datatracker.ietf.org/doc/html/rfc7595#section-8)]. I will add this to development guidelines.

Reply:
> While `urn:x-rdflib:michel` is significantly better than `urn:michel`, we should rather be using example URN namespace `urn:example:` [[IETF RFC 6963](https://datatracker.ietf.org/doc/html/rfc6963)] or even the `example:` URI scheme [[IETF RFC 7595](https://datatracker.ietf.org/doc/html/rfc7595#section-8)]. I will add this to development guidelines.

It's still work-in-progress ... I made it all consistent in an effort to try and clarify the parser-wrapping issue and to resolve three or four test failures caused by mismatches in the namespace output. I'll switch them over to `urn:exanple`.